_posts/2021-03-03-developers-preview-survey-results.md

```diff
@@ -0,0 +1,328 @@
+
+Our aim was to maximise our reach towards potential participants 
+within the Scala community. To achieve this, the main channels of distribution were: 
+the above-mentioned blog article, Twitter (scala_lang), the Contributors forum, various Slack channels, and mailing lists.
+
+Since 14th January 2021 and even until publishing these results, answers are 
+still coming in. Here’s the graph exporting the timestamp of the answer activity:
+![Answers' activity](/resources/img/blog/developers-preview-survey-results/answers-activity.svg)
+
+We have received, to this date (11th February 2021), 578 answers, 
+including a total of 413 comments across 5 comment sections: 4 distinct categories and 1 free.
+
+We identify several threats to validity:
+* **Format of questions:** measuring excitement may not be representative in those cases 
+  where a person is not familiar with a certain feature, tool, and such;
+* **During the collecting period (timespan):** there were a couple of discussions about additional 
+  changes introduced, which could influence the “mood”, especially in answers from the second half of January;
+* **Sample:** distribution methods were as wide as we had at our disposal, 
+  but there’s always a possibility that a certain “group” of users are not represented.
+
+With all that said, our approach to analyse the results is as follows:
+* Presenting the quantifiable data
+* Categorizing comments to provide further insight into the topics discussed; 
+  the purpose of the comment, such as: ‘advice’, ‘concern’, ‘request’; 
+  overall tone: either positive or negative; to prioritise what needs addressing/fixing 
+  before RC1 or at later time, e.g. ‘critical’ or ‘good to have’.
+* Cross-referencing comments with other sources on the same topic.
+* Finally, creating a list of important topics that need to be addressed.
+
+## 2 - Presenting data
+### Features
+![Feature result survey](/resources/img/blog/developers-preview-survey-results/features-survey.svg)
+We asked participants the question “How excited are you about the upcoming Scala 3 language Features”, 
+the results of which are displayed in the above chart. 
+It is sorted in descending order based on the counts for the Excited response.
+
+We can see from the given responses that the features with the most excitement include Enums (**85.8% Excited**), 
+Union/Intersection Types (**83.9% Excited**), Extension Methods (**79.4% Excited**), 
+and Type Class Derivation (**74.0% Excited**).
+
+The features with the most responses were Enums (12 unanswered) and Optional Braces (12 unanswered); 
+from this we can infer that respondents were most familiar with these changes.
+![Optional braces](/resources/img/blog/developers-preview-survey-results/optional-braces.svg)
+
+The most controversial feature found in the responses was Optional Braces, 
+with 28.2% Neutral responses being the smallest proportion of answered responses.
+
+**Potential issues with the poll:**
```

Comment:
Is this a section title? Maybe use the appropriate markdown syntax instead of marking the whole line as bold?

Reply:
It's not a title. It's part of presenting data for features.